BAZProject/SearchMovie/SearchMovieInteractorImp.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SearchMovieInteractor.swift
+//  BAZProject
+//
+//  Created 1030364 on 17/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class SearchMovieInteractorImp: SearchMovieInteractor {
```

Comment:
Recuerda marcar tus clases como `final`
```suggestion
final class SearchMovieInteractorImp: SearchMovieInteractor {
```